### PR TITLE
🐛 Fix `CLUSTERS_KEEPER_TASK_INTERVAL` env var

### PR DESCRIPTION
**Pull request by giancarloromeo**
## What do these changes do?
Fix the `CLUSTERS_KEEPER_TASK_INTERVAL` validation (not anymore necessary).

Q: Why didn't we catch it locally?
A: The `ops-config`s were updated, but not the `.env-devel` file.

```
pydantic_core._pydantic_core.ValidationError: 1 validation error for ApplicationSettings

CLUSTERS_KEEPER_TASK_INTERVAL

  Value error, invalid literal for int() with base 10: '00:00:30' [type=value_error, input_value='00:00:30', input_type=str]

    For further information visit https://errors.pydantic.dev/2.9/v/value_error
```

## Dev-ops checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

[MR-1003](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/merge_requests/1003)

### CODE REVIEW

File: .env-devel [hunk 1]
```diff
@@ -59,7 +59,7 @@ CLUSTERS_KEEPER_EC2_INSTANCES_PREFIX=""
 CLUSTERS_KEEPER_LOGLEVEL=INFO
 CLUSTERS_KEEPER_MAX_MISSED_HEARTBEATS_BEFORE_CLUSTER_TERMINATION=5
 CLUSTERS_KEEPER_PRIMARY_EC2_INSTANCES=null
-CLUSTERS_KEEPER_TASK_INTERVAL=30
+CLUSTERS_KEEPER_TASK_INTERVAL=00:00:30
```
Comment (giancarloromeo) [inline .env-devel:62]: Yes, https://git.speag.com/oSparc/osparc-ops-deployment-configuration/ is updated.

File: services/clusters-keeper/src/simcore_service_clusters_keeper/core/settings.py [hunk 2]
```diff
@@ -381,9 +381,7 @@ def LOG_LEVEL(self) -> LogLevel:  # noqa: N802
     def _valid_log_level(cls, value: str) -> str:
         return cls.validate_log_level(value)
 
-    @field_validator(
-        "CLUSTERS_KEEPER_TASK_INTERVAL", "SERVICE_TRACKING_HEARTBEAT", mode="before"
-    )
+    @field_validator("SERVICE_TRACKING_HEARTBEAT", mode="before")
```
Comment (sanderegg) [inline services/clusters-keeper/src/simcore_service_clusters_keeper/core/settings.py:384]: what about SERVICE_TRACKING_HEARTBEAT?

**Review by sanderegg — APPROVED** [review]

File: .env-devel (hunk 1 above)
Comment (pcrespov) [reply .env-devel:62]: We usually add those MR links  in the PR here
image

**Review by pcrespov — APPROVED** [review]
thx

File: .env-devel (hunk 1 above)
Comment (giancarloromeo) [reply .env-devel:62]: Yup, I have to find them... the files in ops-config were updated already.

**Review by mrnicegyu11 — APPROVED** [review]
looks good to me thanks a lot

File: .env-devel (hunk 1 above)
Comment (giancarloromeo) [reply .env-devel:62]: Already put on description. https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/merge_requests/1003

**Pull request merged**